plugin/kubernetes/object/informer.go

```diff
@@ -20,8 +20,10 @@ func NewIndexerInformer(lw cache.ListerWatcher, objType runtime.Object, h cache.
 	return clientState, cache.New(cfg)
 }
 
-// DefaultProcessor is a copy of Process function from cache.NewIndexerInformer except it does a conversion.
-func DefaultProcessor(convert ToFunc) ProcessorBuilder {
+type recordLatencyFunc func(interface{})
```

Comment:
empty interface? why?

Reply:
Ah, yes, I think I could tighten it down to `meta.Object`, which is implemented by all k8s api objects (e.g. Service, Pod, Endpoints, )...

Reply:
> I could tighten it down to meta.Object

Done in 8aabd9c